<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Peripheral</class>
 <widget class="QDialog" name="Peripheral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1643</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="cell_characteristics">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>40</y>
     <width>261</width>
     <height>451</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QListWidget" name="celllist_widget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="my_qlabel" name="image_label">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>40</y>
     <width>960</width>
     <height>701</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="openfile_button">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>270</y>
     <width>81</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/open_icon.png</normaloff>../icon/open_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="save_celltypes_button">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>160</y>
     <width>81</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/save_icon.png</normaloff>../icon/save_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="my_qlabel" name="cell_label">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>40</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="selection_type">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>230</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Manuel Selection</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>530</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Prediction :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Result_value">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>530</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>my_qlabel</class>
   <extends>QLabel</extends>
   <header location="global">my_qlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
